<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MPRViewer</class>
 <widget class="QMainWindow" name="MPRViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MPRViewer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>22</width>
    <height>22</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="listWidget"/>
       </item>
       <item>
        <widget class="QPushButton" name="loadFileButton">
         <property name="text">
          <string>Load .mhd raw file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget_2"/>
       </item>
       <item>
        <widget class="QPushButton" name="loadVTKButton">
         <property name="text">
          <string>Load .vtk file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QVTKWidget" name="view2" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QVTKWidget" name="view4" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="QVTKWidget" name="view3" native="true"/>
      </item>
      <item row="0" column="0">
       <widget class="QVTKWidget" name="view1" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>13</width>
        <height>17</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignLeft">
     <widget class="QFrame" name="frame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QCheckBox" name="resliceModeCheckBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>20</y>
         <width>121</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Oblique Reslice</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="thickModeCheckBox">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Thick Slab</string>
       </property>
      </widget>
      <widget class="QGroupBox" name="blendModeGroupBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>161</width>
         <height>91</height>
        </rect>
       </property>
       <property name="title">
        <string>Blend mode</string>
       </property>
       <widget class="QRadioButton" name="radioButton_Min">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Min Intensity Blend</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_Max">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>141</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Max Intensity Blend</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_Mean">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Mean Blend</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="resetButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>141</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="icon">
        <iconset resource="MPRViewerIcons.qrc">
         <normaloff>:/common/data/icons/open.png</normaloff>:/common/data/icons/open.png</iconset>
       </property>
      </widget>
      <widget class="QFrame" name="frame_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>250</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="AddDistance3Button">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Add Distance On View 3</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="addCircumferenceButton">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>300</y>
         <width>181</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Add circumference on View 3</string>
       </property>
      </widget>
      <widget class="QLabel" name="surfaceAreaLabel">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>390</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Area</string>
       </property>
      </widget>
      <widget class="QPushButton" name="calculateCircumferenceButton">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>151</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Calculate surface area</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>390</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Area:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Diameter:</string>
       </property>
      </widget>
      <widget class="QLabel" name="diameterLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>420</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Diameter</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>390</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mm2</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>420</y>
         <width>55</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>mm2</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionOpenFile">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/fileopen.png</normaloff>:/Icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open File...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/print.png</normaloff>:/Icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/help.png</normaloff>:/Icons/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>:/Icons/filesave.png</normaloff>:/Icons/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="MPRViewerIcons.qrc"/>
 </resources>
 <connections/>
</ui>
